var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -441,6 +453,22 @@ def cmake_args(self):
         # In other words, the mapping between package variants and the
         # GMX CMake variables is often non-trivial.
 
+        gmx_cc = spack_cc
+        gmx_cxx = spack_cxx
+        if "+rocm" in self.spec:
+            # The ROCm version requires the ROCm LLVM installation
```

Comment:
I think this commend is incorrect: modern mainline Clang works fine (GROMACS does recommend amdclang, but it's just a recommendation to avoid version hell). The code below is correct as far as I can tell, it's just the comment that's off.

Reply:
Removed the comment.